WindowsServerDocs/manage/windows-admin-center/extend/guides/add-module.md

```diff
@@ -29,7 +29,7 @@ If you are new to Angular, it is highly recommended that you read the documentat
 Open a command prompt, change directory to \src\app in your project, then run the following commands, replacing ```{!ModuleName}``` with your module name (spaces removed):
```

Comment:
Change this line too, just to make sure that it conforms with the suggested change below:
```suggestion
Open a command prompt, change directory to .\src\app in your project, then run the following commands, replacing ```{!ModuleName}``` with your module name (spaces removed):
```